rootston/keyboard.c

```diff
@@ -32,6 +32,11 @@ static void keyboard_binding_execute(struct roots_keyboard *keyboard,
 		if (keyboard->input->last_active_view != NULL) {
 			view_close(keyboard->input->last_active_view);
 		}
+	} else if (strcmp(command, "next window") == 0) {
```

Comment:
I know we don't have a parser, but `next_window` seems a bit nicer to me so we have one command per word.

Reply:
Oh_yeah,_totaly.